utils/Cargo.toml

```diff
@@ -38,4 +38,10 @@ server = [
 
 [dev-dependencies]
 serde_json = "1.0"
-tokio = { version = "1", features = ["macros", "rt"] }
\ No newline at end of file
+env_logger = "0.9"
+tokio = { version = "1", features = ["macros", "rt"] }
+# TODO: (dp) seems like the macro doesn't work with the component crates, but users must import `jsonrpsee`?
+# jsonrpsee-proc-macros = { path = "../proc-macros" }
+# jsonrpsee-types = { path = "../types" }
+# jsonrpsee-ws-server = { path = "../ws-server" }
```

Comment:
@niklasad1 Is this a known gotcha with the proc macro? I couldn't make the proc macro work without importing the full `jsonrpsee` crate.

Reply:
ah right, I see that's expected.

Can't you move this test to integration-tests/proc macros instead?